docs/architecture/grpc-for-wcf-developers/encryption.md

```diff
@@ -6,23 +6,23 @@ ms.date: 09/02/2019
 
 # Encryption and network security
 
-WCF's network security model is extensive and complex, including transport-level security using HTTPS or TLS-over-TCP, and message-level security using the WS-Security specification to encrypt individual messages.
+The network security model for Windows Communication Foundation (WCF) is extensive and complex. It includes transport-level security by using HTTPS or TLS-over-TCP, and message-level security by using the WS-Security specification to encrypt individual messages.
 
-gRPC leaves secure networking to the underlying HTTP/2 protocol, which can be secured using regular TLS certificates.
+gRPC leaves secure networking to the underlying HTTP/2 protocol, which you can secure by using regular TLS certificates.
 
-Web browsers insist on using TLS connections for HTTP/2, but most programmatic clients, including .NET's `HttpClient`, can use HTTP/2 over unencrypted connections. `HttpClient` *does* require encryption by default, but you can override this using an <xref:System.AppContext> switch.
+Web browsers insist on using TLS connections for HTTP/2, but most programmatic clients, including .NET's `HttpClient`, can use HTTP/2 over unencrypted connections. `HttpClient` does require encryption by default, but you can override this by using an <xref:System.AppContext> switch.
 
 ```csharp
 AppContext.SetSwitch("System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport", true);
 ```
 
-For public APIs, you should always use TLS connections and provide valid certificates for your services from a proper SSL authority. [LetsEncrypt](https://letsencrypt.org) provide free, automated SSL certificates, and most hosting infrastructure today supports the LetsEncrypt standard with common plug-ins or extensions.
+For public APIs, you should always use TLS connections, and provide valid certificates for your services from a proper SSL authority. [LetsEncrypt](https://letsencrypt.org) provides free, automated SSL certificates, and most hosting infrastructure today supports the LetsEncrypt standard with common plug-ins or extensions.
 
 For internal services across a corporate network, you should still consider using TLS to secure network traffic to and from your gRPC services.
 
-Communication between microservices in a cluster like Kubernetes or Docker Swarm is in general automatically encrypted by the container networking layer, so implementing TLS in services running exclusively in such a cluster isn't necessary. There will be more on this subject in the "Service Mesh" section of the next chapter.
+Communication between microservices in a cluster like Kubernetes or Docker Swarm is, in general, automatically encrypted by the container networking layer. Implementing TLS in services running exclusively in such a cluster isn't necessary.
```

Comment:
I'm not sure this statement is technically accurate? @jamesnk do you know?

Reply:
I'm not sure about that statement either. Citation needed.

Reply:
I'd cut anything you are not sure about.

Reply:
@shirhatti @JamesNK can you provide a suggestion here of what the text should look like?

Reply:
I don't think this statement is accurate at all: https://stackoverflow.com/a/45456438/11829

I did find this - https://docs.docker.com/ee/ucp/kubernetes/kubernetes-network-encryption/ - but it seems like a feature specific to a product.

IMO remove this paragraph, and rephrase the next paragraph to discuss how to do TLS in Kubernetes.

@rynowak You have more Kubernetes experience. Speak up if you have any insight here.